### PR TITLE
OPENCAST-2647 - refactor and config update

### PR DESCRIPTION
**Pull request by NgoniChoga**

### CODE REVIEW

**Review by TurRil — APPROVED** [review]

File: config-dist.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+DIGEST_AUTH = {
+    "username": "my_usename",
+    "password": "my_password"
+}
+
+BASICAUTH_USERS_LIST = {
+    "username": "password"
+}
+
+CAPTURE_AGENT_URL = "https://opencast.org/capture-admin/agents.json"
```
Comment (smarquard) [inline config-dist.py:13]: You could use 
https://stable.opencast.org/ as the demo URL here.

File: jobs/feeds.py [hunk 2]
```diff
@@ -10,24 +10,23 @@
 
 @shared_task
 def get_feeds():
-    commands = []
     all_venues = Venues.objects.all()
     check_and_stop_running_processes()
 
     for venue in all_venues:
         venue_name = venue["venue_name"]
```
Comment (smarquard) [inline jobs/feeds.py:17]: As venue_name is used to construct a path name on the filesystem, best to apply a strict regexp to it e.g. A-Za-z0-9 and ignore venues that don't pass the regexp.

**Review by smarquard — CHANGES_REQUESTED** [review]

File: config-dist.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Do not change this directory path
+DIRECTORY = "sync/static/feeds/"
+
+DIGEST_AUTH = {
+    "username": "my_usename",
```
Comment (smarquard) [inline config-dist.py:5]: Use admin / opencast here for username/pwd which would make it work with stable.opencast.org by default

File: config-dist.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "password": "my_password"
+}
+
+BASICAUTH_USERS_LIST = {
```
Comment (smarquard) [inline config-dist.py:9]: Remove the basic auth from this app as we will handle it in nginx.

**Pull request merged**